CONTRIBUTING.md

```diff
@@ -1,59 +1,91 @@
+- [domumentation](https://github.com/CircuitVerse/CircuitVerse/labels/documentation) labeled issues are meant to have typo errors in documentation.
+- [Simulator](https://github.com/CircuitVerse/CircuitVerse/labels/simulator) issues are meant specifically in the scope of circuitverse simulator.
+- [platform](https://github.com/CircuitVerse/CircuitVerse/labels/platform) labeled issues are meant specifically in the scope of circuitverse website.
+- `difficulty: easy` issues are usually confined to isolated areas of existing code.
+- `difficulty: medium` issues sometimes entail new features and might affect a significant area of the codebase, but aren't overly complex.
+- `difficulty: hard` issues are typically far-reaching, and might need architecture decisions during implementation. This label might also denote highly complex issues.
+- `duplicate` labelled issues are meant to be already existing issue in the repository.
+- `priority: less` labelled issues are meant to have priority comparatavily lesser than other issues.
+- `priority: medium` labelled issues are meant to have priority comparatively intermediate than other issues.
+- `priority: high` labelled issues are meant to have highest priority and need to fix as soon as possible.
+- `help wanted` labelled issues are meant to get help from core-team.
+
+## Creating an issue.
+
+- Check if the issue you are going to propose is not duplicate of another issue.
+- Open a new issue according to type i.e., if issue is a bug open a new issue by clicking on `Get Started` in the scope of `Bug Report`.
+- Give a precise and meaningful name of the issue.
+- Describe your issue as good as possible that may ease the process of issue-reviewing by core-team.
+
+## How to contribute
+
+1. Fork the project and clone it to your local machine. Follow the [setup](https://github.com/CircuitVerse/CircuitVerse/blob/master/SETUP.md) guideline.
+2. Always take a pull from the remote repository to your master branch to keep it at par with the main project(updated repository).
+        
+        git pull upstream master
+        
+3. Create a branch. For example, if you are going to work on issue number #44 (which is, say, a new feature for ‘forgot password’ management):
 
         git checkout -b forgot-password#44
 
     This both creates and checks out that branch in one command.  
     The feature name should provide a (short) description of the issue.
 
-2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
-   
-3. Develop the feature and push it to your local repo.
-   
-4. Create Pull Request (PR). Make sure to comment the issue that your PR is supposed to solve.
+4. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+5. Develop the feature and push it to your local repo.
```

Comment:
```suggestion
5. Commit your changes and push it to your fork of the repository.
```
Local repo can be confused with repository on local machine.

Reply:
Done